cli/azd/internal/telemetry/fields/fields.go

```diff
@@ -47,6 +47,12 @@ const (
 	//
 	// See EnvDesktop for complete set of values.
 	ExecutionEnvironmentKey = attribute.Key("execution.environment")
+
+	// Installer used to install the application. Set in .installed-by.txt file
+	// located in the same folder as the executable.
+	//
+	// Example: "msi", "brew", "choco", "rpm", "deb"
+	InstalledByKey = attribute.Key("process.executable.installer")
```

Comment:
There is a little bit of conundrum, but conceding some level of defeat, I think `service.installer` might be the best out of naming options.

We currently set `service.name` and `service.version` as part of the required opentelemetry fields schema. And I would love to group "version" and "installer" under the same bucket.

Reply:
I have no strong opinions on names here